pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -58,6 +58,10 @@ For more information, see the [air-gapped bundle](../../install/air-gapped) docu
 
 The unified DKP user interface provides a smooth experience independent of where you start your journey. Both DKP Essential and DKP Enterprise customers will have similar experiences in the User Interface, with DKP Enterprise customers gaining access to additional features and benefits simply by entering their DKP Enterprise [license key](../../licensing/).
 
+### Kaptain AI/ML, D2iQ’s AI/ML offering
+
+For better integration with DKP 2.2 you can launch Kaptain as a catalog application, and also support other platforms such as Amazon AWS, AKS, and Microsoft Azure EKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.
```

Comment:
```suggestion
For better integration with DKP 2.2 you can launch Kaptain as a catalog application, and also support other platforms such as Amazon AWS EKS, and Microsoft Azure AKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.
```

Reply:
lol good catch! I could say I just copied that text from somewhere else, but that'd just be me defending myself 🙈 😂 

https://github.com/mesosphere/dcos-docs-site/pull/4206/commits/5dd71aa8b96068619fd9335ab93addac4a0afc3d